webpack.config.js

```diff
@@ -0,0 +1,60 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
So let's a quick little clean up here
1.  let's call this file `webpack.config.common.js`
1.  move just the `devServer` configuration into a new file called `webpack.config.develop.js` 
1.  have this develop config use *webpackMerge* with the common config